src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html

```diff
@@ -1,85 +1,101 @@
-          <div class="cd-col-form-input">
-            <select class="form-select"
-                    id="name"
+    <!-- Name -->
+    <div cdsRow
+         class="form-item form-item-append">
+      <div cdsCol>
+        <cds-select id="name"
                     formControlName="name"
-                    name="name">
-              <option [ngValue]="null"
-                      i18n>-- Select an attribute to match against --</option>
-              <option *ngFor="let attribute of nameAttributes"
-                      [value]="attribute">
-                {{ attribute }}
-              </option>
-            </select>
-            <span class="help-block"
-                  *ngIf="form.showError('name', formDir, 'required')"
-                  i18n>This field is required!</span>
-          </div>
-        </div>
-
-        <!-- Value -->
-        <div class="form-group row">
-          <label class="cd-col-form-label required"
-                 for="value"
-                 i18n>Value</label>
-          <div class="cd-col-form-input">
-            <input id="value"
-                   (focus)="valueFocus.next($any($event).target.value)"
-                   (click)="valueClick.next($any($event).target.value)"
-                   class="form-control"
-                   type="text"
-                   [ngbTypeahead]="search"
-                   formControlName="value"
-                   #instance="ngbTypeahead">
-            <span *ngIf="form.showError('value', formDir, 'required')"
-                  class="help-block"
-                  i18n>This field is required!</span>
-          </div>
+                    label="Name"
+                    i18n-label
+                    placeholder="Select a name..."
+                    i18n-placeholder
+                    [invalid]="form.controls.name.invalid && form.controls.name.dirty"
+                    [invalidText]="nameError"
+                    helperText="This name will define and control the matcher settings"
+                    i18n-helperText>
+        <option [ngValue]="null"
+                i18n>-- Select an attribute to match against --</option>
+        <option *ngFor="let attribute of nameAttributes"
+                [value]="attribute">
+        {{ attribute }}
+        </option>
+        </cds-select>
+        <ng-template #nameError>
+          <span *ngIf="form.showError('name', formDir, 'required')"
+                class="invalid-feedback">
+            <ng-container i18n>This field is required.</ng-container>
+          </span>
+        </ng-template>
+      </div>
+    </div>
+    <div cdsRow
+         class="form-item form-item-append">
+      <!-- Value -->
+      <div cdsCol>
+        <cds-combo-box label="Value"
+                       cdsRequiredField="Value"
+                       for="value"
+                       formControlName="value"
+                       id="value"
+                       placeholder="Select Values..."
```

Comment:
This only adds confusion because you can only select one value but it's telling you to select multiple.


```suggestion
```